### DISCUSSION TITLE
Unlisted GitHub Repos

### DISCUSSION THREAD

**Discussion by jra457** [community/community#113466] [Repositories]
### Select Topic Area

Product Feedback

### Body

Having the ability to set repositories as private, and then sharing the link to people who you want to be able to access the repository without making it public would be nice to have.

I am currently a student and would like to compile all of my code that I have worked on in school to one repository, and include it on my Resume. However, coding homework and assignments are often recycled every couple of years, and I have concerns about uploading what would essentially be a cheat sheet for these assignments to the web. 

A common suggestion is to simply add collaborators to a private repository so they can view the, but when it comes to applying for internships, co-ops, or jobs, it would be uncommon for a recruiter to email you asking to be added to your GitHub repository. It would also be awkward to include on your Resume to "Email me to be added to my GitHub repository." 

Thus, being able to have a link that allows users to access the repository without specifically adding them would allow students to share repositories on their Resumes, without risking academic honesty and integrity.

**Comment by look** [comment]
👋 You could use a [Gist](https://gist.github.com/) for this. They are full-fledged repositories, but the sharing is like what you describe: a secret link.
